packages/data-stores/src/user/index.ts

```diff
@@ -11,21 +11,21 @@ import reducer, { State } from './reducer';
 import * as actions from './actions';
 import * as resolvers from './resolvers';
 import * as selectors from './selectors';
-import createControls from './controls';
+import { WpcomClientCredentials, controls } from '../wpcom-request-controls';
 import { DispatchFromMap, SelectFromMap } from '../mapped-types';
-import { WpcomClientCredentials } from '../utils';
 
 export * from './types';
 export { State };
 
 let isRegistered = false;
-export function register( clientCreds: WpcomClientCredentials ): typeof STORE_KEY {
+export function register(
+	/* @TODO: deal with creds */ clientCreds: WpcomClientCredentials
+): typeof STORE_KEY {
 	if ( ! isRegistered ) {
-		const controls = createControls( clientCreds );
 		isRegistered = true;
 		registerStore< State >( STORE_KEY, {
 			actions,
-			controls,
+			controls: controls as any,
```

Comment:
It doesn't like the controls type right now, maybe just remove the `as const` cast from controls definition.

Reply:
Removing the `as const` from the controls object isn't enough to make the types match so I've left this as is.